### ISSUE TITLE
Some firejail wrapped binaries cannot be started

### ISSUE THREAD

**Issue by wentasah** [NixOS/nixpkgs#214176]
### Describe the bug

Having the following in `configuration.nix`:

```nix
{
  programs.firejail = {
    enable = true;
    wrappedBinaries = {
      slack = "${pkgs.slack}/bin/slack";
    };
  };
}
```

and running `slack` results in:

```
Cannot start application: Permission denied
```

### Steps To Reproduce

See above.

### Expected behavior

The `slack` application starts.

### Additional context

This is most likely caused by firejail bugs: https://github.com/netblue30/firejail/issues/5605 and https://github.com/netblue30/firejail/issues/5605. It happens with
```
$ firejail --version
firejail version 0.9.72
```

A workaround is to start the wrapped application without `--` as used in the wrapper. For example, when the wrapper contains:
```sh
exec /run/wrappers/bin/firejail  -- /nix/store/p5il5hglrlvhsjxw4b28yw3h85ammr4r-slack-4.29.149/bin/slack "$@"
```
run :
```sh
firejail /nix/store/p5il5hglrlvhsjxw4b28yw3h85ammr4r-slack-4.29.149/bin/slack
```

I think, we should wait for upstream fix to resolve this issue. I hope this post might help other people who encounter the problem.

### Notify maintainers

@peterhoeg

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.90, NixOS, 23.05 (Stoat), 23.05.20230131.2b4ca18`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.2`
 - channels(root): `""`
 - channels(wsh): `""`
 - nixpkgs: `/home/wsh/nix/nixpkgs`
```

**Comment by revuwa** [comment]
Just wanted additionally point to the comments of PR: https://github.com/NixOS/nixpkgs/pull/211245

**Comment by wentasah** [comment]
Fixed long time ago by #216778.
